.clang-format

```diff
@@ -0,0 +1,10 @@
+BasedOnStyle: LLVM
+IndentWidth: 4
+UseTab: Never
+BreakBeforeBraces: Linux
+AlignAfterOpenBracket: Align
+AllowShortIfStatementsOnASingleLine: true
+AllowAllArgumentsOnNextLine: false
+BinPackArguments: false
+BinPackArguments: false
```

Comment:
Unintentionally doubled?

Reply:
Ack, the other one should have been the other BinPack<something>...